docker/ludwig/Dockerfile

```diff
@@ -9,7 +9,7 @@
 #   model serving
 #
 
-FROM python:3.7.12-slim
+FROM python:3.8.12-slim
```

Comment:
We should check what the latest patch version is on DockerHub for 3.8

Reply:
seems like latest is 3.8.12

https://hub.docker.com/_/python